src/LibraBFT/Impl/Consensus/BlockStorage/BlockTree-AST.agda

```diff
@@ -0,0 +1,103 @@
+{- Byzantine Fault Tolerant Consensus Verification in Agda, version 0.9.
```

Comment:
I am not reviewing this file:

- I think I did a light review of these changes when this code was contained in the `BlockTree.agda` file.

So it is just separated into its own file now, right?

Reply:
> Approved. I think we should (in a subsequent PR) separate the last pieces of Haskell imported by Dijkstra.AST into its own top level module (like Haskell).

We can do that.  The reason I didn't do it already was that I wanted to seek input on appropriate organisation and naming.  If you or @cwjnkins have any suggestions, maybe we can do it before the paper deadline, otherwise it can wait.

Reply:
> I am not reviewing this file:
> 
> * I think I did a light review of these changes when this code was contained in the `BlockTree.agda` file.
> 
> So it is just separated into its own file now, right?

Yes, correct.